bokeh/server/application_context.py

```diff
@@ -137,7 +137,7 @@ def run_unload_hook(self):
         self.server_context._remove_all_callbacks()
 
     @gen.coroutine
-    def create_session_if_needed(self, session_id):
+    def create_session_if_needed(self, session_id, request_arguments=None):
```

Comment:
This same function is called by the WS view handler, which has no request arguments to pass, hence the default `None` parameter. Not sure if there's a cleaner way to go here.